lib/rules/media-query-no-invalid/README.md

```diff
@@ -62,3 +62,32 @@ The following patterns are _not_ considered problems:
 ```css
 @media (color) and ((hover) or (width)) {}
 ```
+
+## Optional secondary options
+
+### `ignoreFunctions: ["/regex/", /regex/, "string"]`
+
+Ignore the specified functions.
+
+Given:
+
+```json
+["theme", "/^get.*$/"]
+```
+
+The following patterns are _not_ considered problems:
+
+<!-- prettier-ignore -->
+```css
+@media (min-width: theme(screens.md)) {}
+```
+
+<!-- prettier-ignore -->
+```css
+@media (min-width: getDefaultWidth()) {}
+```
+
+<!-- prettier-ignore -->
+```css
+@media (max-width: get-default-width()) {}
+```
```

Comment:
[question] Are both `getDefaultWidth` and `get-default-width` still necessary? Either one seems enough to me to show an example for `/^get.*$/`. 🤔

Reply:
Sorry, I misunderstood for your [previous suggestion](https://github.com/stylelint/stylelint/pull/8060#discussion_r1832678871).
Fixed this commit.
[remove redundant examples](https://github.com/stylelint/stylelint/pull/8060/commits/c3a89d71d005f9e6652f32408654261236b2e0b0)